resources/assets/js/annotations/annotatorContainer.vue

```diff
@@ -571,8 +571,27 @@ export default {
         dismissCrossOriginError() {
             this.crossOriginError = false;
         },
-        handleInvalidPolygon() {
-            Messages.danger(`Invalid shape. Polygon needs at least 3 non-overlapping vertices.`);
+        handleInvalidShape(shape) {
+            let count;
+            switch(shape){
+                case 'Circle':
+                    Messages.danger('Invalid shape. Circle needs non-zero radius');
+                    return;
+                case 'LineString':
+                    shape = 'Line'
+                    count = 2;
+                    break;
+                case 'Polygon':
+                    count = 'at least 3';
+                    break;
+                case 'Rectangle':
+                case 'Ellipse':
+                    count = 4;
+                    break;
+                default:
+                    return;
+            }
+            Messages.danger(`Invalid shape. ${shape} needs ${count} non-overlapping vertices.`);
```

Comment:
I would phrase this a little simpler for the non-technical users:

```suggestion
            Messages.danger(`Invalid shape. ${shape} needs ${count} different points.`);
```